ambiata-p.cabal

```diff
@@ -15,8 +15,9 @@ description:           A set of standard preludes for Ambiata projects. \
 library
   build-depends:
                           base                        >= 4.6        && < 5
-                        , bifunctors                  >= 4.2        && <= 5.3
-                        , deepseq                     >= 1.3        && <= 1.5
+                        , bifunctors                  >= 4.2        && < 5.4
+                        , containers                  >= 0.5        && < 0.6
+                        , deepseq                     >= 1.3        && < 1.5
```

Comment:
Should probably be a separate PR, but I noticed when I added containers that we had `<=` on some upper bounds so I've corrected it.

Note that the latest version of `bifunctors` is `5.3`, and the latest version of `deepseq` is `1.4.2.0`. So the bound we had on `deepseq` was incorrect (my bad actually).